src/view/dataElements/xdmObject.jsx

```diff
@@ -164,38 +285,87 @@ const XdmExtensionView = () => {
       }}
       getSettings={({ values }) => {
         return {
+          sandbox: {
+            name: selectedSandboxMeta ? selectedSandboxMeta.name : null
+          },
```

Comment:
Hey @garyr, it doesn't look like these changes are reflected in the schema defined in `extension.json`: https://github.com/adobe/reactor-extension-alloy/blob/master/extension.json#L258-L272

Reply:
BTW, you won't be able to make `sandbox` a new required field in the schema, because it would be considered backward-incompatible (https://developer.adobelaunch.com/extensions/reference/backwards-compatibility/). In other words, there are already settings objects in the database that don't have `sandbox` defined, so you can't change the schema to say that it's required. You may have already handled this (I haven't looked yet), but you'll need to make sure than you can load existing settings objects that don't contain the `sandbox` attribute into the view.

Reply:
Thanks. I will look into this...